css/index.css

```diff
@@ -66,6 +66,44 @@ p {
     line-height: 1.4;
 }
 
+/*Header*/
+header nav {
```

Comment:
You can set the nav to the same width as the rest of the site so it won't spread out across the whole width of the screen.